medical_lab/models/medical_lab.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012-2013 Federico Manuel Echeverri Choux
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class MedicalLab(models.Model):
+    _name = "medical.lab"
```

Comment:
Maybe the data model should be:
- medical.lab.analysis: for this object which would contain many Tests
  - medical.lab.test: the list of tests to be performed in this Analysis (which could be renamed from medical.lab.test.result)

If we follow this, we might also want to rename `medical.lab.test.type` to `medical.lab.analysis.type`.

I understand that it might not be your intention to do this changes at this time, but if not done now I don't know when it can happen.